DB/src/main/java/io/deephaven/db/tables/utils/csv/InferenceSpecs.java

```diff
@@ -0,0 +1,294 @@
+                Parser.STRING,
+                Parser.BYTE,
+                Parser.SHORT,
+                Parser.INT,
+                Parser.FLOAT,
+                Parser.CHAR)
+                .build();
+    }
+
+    /**
+     * The "standard" inference, does not parse floats or bytes.
+     *
+     * <p>
+     * Contains the following parsers:
+     *
+     * <ul>
+     * <li>{@link Parser#INSTANT}</li>
+     * <li>{@link Parser#SHORT}</li>
+     * <li>{@link Parser#INT}</li>
+     * <li>{@link Parser#LONG}</li>
+     * <li>{@link Parser#DOUBLE}</li>
+     * <li>{@link Parser#BOOL}</li>
+     * <li>{@link Parser#CHAR}</li>
+     * <li>{@link Parser#STRING}</li>
+     * <li>{@link Parser#BYTE}</li>
+     * <li>{@link Parser#FLOAT}</li>
+     * </ul>
+     *
+     * Uses the default {@link #onNullParser()}.
+     *
+     * <p>
+     * Note: the byte and float parsers are only relevant when the appropriate {@link #limitToType(Type)} is invoked.
+     *
+     * @return the standard inference
+     */
+    public static InferenceSpecs standard() {
+        return builder().addParsers(
+                Parser.INSTANT,
+                Parser.SHORT,
+                Parser.INT,
+                Parser.LONG,
+                Parser.DOUBLE,
+                Parser.BOOL,
+                Parser.CHAR,
+                Parser.STRING,
+                Parser.BYTE,
+                Parser.FLOAT)
+                .build();
+    }
+
+    /**
+     * The standard parsers with additional {@link java.time.Instant}-based parsing.
+     *
+     * <p>
+     * Contains the following parsers:
+     *
+     * <ul>
+     * <li>{@link Parser#INSTANT}</li>
+     * <li>{@link Parser#INSTANT_DB}</li>
+     * <li>{@link Parser#epochMilliAndMicroParsers21stCentury(Parser)}, with {@link Parser#LONG}</li>
+     * <li>{@link Parser#SHORT}</li>
+     * <li>{@link Parser#INT}</li>
+     * <li>{@link Parser#LONG}</li>
+     * <li>{@link Parser#DOUBLE}</li>
+     * <li>{@link Parser#BOOL}</li>
+     * <li>{@link Parser#CHAR}</li>
+     * <li>{@link Parser#STRING}</li>
+     * <li>{@link Parser#BYTE}</li>
+     * <li>{@link Parser#FLOAT}</li>
+     * </ul>
+     *
+     * Uses the default {@link #onNullParser()}.
+     *
+     * <p>
+     * Note: the byte and float parsers are only relevant when the appropriate {@link #limitToType(Type)} is invoked.
+     *
+     * @return the standard times inference
+     */
+    public static InferenceSpecs standardTimes() {
+        final List<Parser<Instant>> parsers = Parser.epochMilliAndMicroParsers21stCentury(Parser.LONG);
+        return builder().addParsers(
+                Parser.INSTANT,
+                Parser.INSTANT_DB,
+                parsers.get(0),
+                parsers.get(1),
+                Parser.SHORT,
+                Parser.INT,
+                Parser.LONG,
+                Parser.DOUBLE,
+                Parser.BOOL,
+                Parser.CHAR,
+                Parser.STRING,
+                Parser.BYTE,
```

Comment:
I'm not in love with making BYTE ignored like this, but you may have a good point about it

Reply:
We might actually want to not parse to SHORT as well... I'm not sure...